pkg/internal/checkup/trex/configfiles.go

```diff
@@ -158,3 +159,12 @@ ip_telco1 = '10.1.1.1'
 		c.DPDKWestMacAddress,
 	)
 }
+
+func (c Config) GenerateExecutionScript() string {
+	sb := strings.Builder{}
+
+	sb.WriteString("#!/usr/bin/env bash\n")
```

Comment:
what about licence comment?

Reply:
I'll add it in a follow-up PR if that it OK with you.